cli/src/commands/working-groups/verifyValidator.ts

```diff
@@ -0,0 +1,39 @@
+import { flags } from '@oclif/command'
+import WorkingGroupsCommandBase from '../../base/WorkingGroupsCommandBase'
+import { WorkingGroups } from '../../Types'
+
+export default class MembershipValidatorAccountCommand extends WorkingGroupsCommandBase {
+  static description = 'Membership lead/worker verifies validator membership profile'
+  static flags = {
+    message: flags.string({
+      required: true,
+      description: 'Memberhip ID of the validator to verify',
```

Comment:
This argument's name is confusing and not aligned with the description of the argument. rename it to something like `memberId`